### PR TITLE
WM-2123: Request callback in submissions to Cromwell

### PR DESCRIPTION
**Pull request by cjllanwarne**
Adds an additional workflow option to activate workflow completion callbacks (see https://github.com/broadinstitute/cromwell/pull/7213/files under `docs/cromwell_features/WorkflowCallback.md`).

Note: doesn't start doing until without also having:
- [X] Merging of https://github.com/broadinstitute/cromwell/pull/7213
- [ ] Terra-helmfile [changes](https://github.com/broadinstitute/terra-helmfile/pull/4634) to set up callback config in CBAS and in Cromwell Runner

### CODE REVIEW

**Review by kpierre13 — APPROVED** [review]
LGTM 👍🏾

File: service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java [hunk 1]
```diff
@@ -811,17 +812,26 @@ void testWorkflowOptionsProperlyConstructed() {
...
+    cbasNetworkConfiguration.setExternalUri("http://localhost:8080");
+    CromwellService localtestService =
+        new CromwellService(localTestClient, cromwellClient, cbasNetworkConfiguration);
 
     // Workflow options should reflect the final workflow log directory.
     // write_to_cache should always be true. read_from_cache should match the provided call caching
     // option.
     String expected =
-        "{\"final_workflow_log_dir\":\"my/final/workflow/log/dir\",\"read_from_cache\":true,\"write_to_cache\":true}";
+        "{\"final_workflow_log_dir\":\"my/final/workflow/log/dir\",\"read_from_cache\":true,\"workflow_callback_uri\":\"http://localhost:8080api/batch/v1/runs/results\",\"write_to_cache\":true}";
```
[the remark this answers is not in the document]
Comment (zykonda) [reply service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java:824]: +1 on this note

[the remark this answers is not in the document]
Comment (cjllanwarne) [reply service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java:824]: Ooh, good catch

**Review by zykonda — APPROVED** [review]
LGTM

**Pull request merged**